app/views/admin/responses/_form.html.erb

```diff
@@ -0,0 +1,47 @@
+<%= form_for [:admin, consultation, consultation_response], url: [:admin, consultation, consultation_response.singular_routing_symbol] do |form| %>
+  <%= render "govuk_publishing_components/components/fieldset", {
+    legend_text: "Published on",
+    heading_size: "s"
+  } do %>
+    <%= render "components/datetime_fields", {
+      prefix: "consultation_#{consultation_response.singular_routing_symbol}",
+      field_name: "published_on",
+      # date_heading: "Published on",
+      date_only: true,
+      year: {
+        value: consultation_response.published_on.year,
```

Comment:
```
`value: consultation_response.published_on&.year,`
```

We need to use a safe navigator here or it blows up when the page is submitted with blank values

Reply:
👍